src/Components/Items/ItemPersistentData.tsx

```diff
@@ -68,9 +68,9 @@ export const ItemPersistentData = ({ item }: Props) => {
             backgroundColor={
               statusColor
                 ? "#" +
-                  statusColor.data?.value
+                  (statusColor.data?.value
                     .find((e) => e.name === item.fields["System.State"])
-                    ?.color.toUpperCase()
+                    ?.color.toUpperCase() ?? "808080")
```

Comment:
please, use colors from `theme`

Reply:
These colors are given by Azure Devops, in case it fails it uses the default azure devops color.